frontend/src/pages/LogsPage/SearchForm/SearchForm.tsx

```diff
@@ -142,16 +142,18 @@ const Search: React.FC<{
 			return
 		}
 
-		getLogsKeyValues({
-			variables: {
-				project_id: project_id!,
-				key_name: activeTerm.key,
-				date_range: {
-					start_date: moment(startDate).format(FORMAT),
-					end_date: moment(endDate).format(FORMAT),
+		if (activeTerm.key !== 'message') {
```

Comment:
Tried to use the enum and the build failed...
https://github.com/highlight/highlight/actions/runs/4359439593/jobs/7621231851

Not sure why since this works for me locally